ops/provision/modules/ec2/main.tf

```diff
@@ -27,15 +27,19 @@ resource "aws_instance" "fcrepo" {
   for_each = {
     prod = "prod"
     demo = "demo"
+    alpha = "alpha"
+    bravo = "bravo"
+    charlie = "charlie"
+    delta = "delta"
   }
 
-  ami                         = data.aws_ami.amazon-linux-2.id
+  ami                         = "ami-02538f8925e3aa27a" # data.aws_ami.amazon-linux-2.id
```

Comment:
☝️ why this change? Do we need to pin to this AMI?

Reply:
the AMI got a new release and I didn't want to blow away and rebuild demo/prod while I was hustling to get this work done. once we've got the rush over, rebuild is fairly simple and relatively safe.

Reply:
👍